infrastructure/resource_group_level/modules/webapps/portal-api.bicep

```diff
@@ -40,19 +40,10 @@ param secureConnectionStrings object = {
   CORS_ORIGIN: 'test-value'
   AZURE_PORTAL_EXPORT_FOLDER: 'test-value'
   AZURE_PORTAL_FILESHARE_NAME: 'test-value'
-  // TODO:FN-749 Confirm *_WORKFLOW_* variables are not needed and can be removed.
```

Comment:
As these were removed in the merge from main, I've removed them too. I'll still double check with FN-749.